src/app/globals.css

```diff
@@ -21,7 +21,7 @@
   --type-title-large: 500 1.375rem / 1.75rem var(--font-bai);
   --type-title-large-bold: 700 1.375rem / 1.75rem var(--font-bai);
```

Comment:
This variable isn’t necessary. `type-title-large font-bold` should have the same effect.